### ISSUE TITLE
Progress while moving files

### ISSUE THREAD

**Issue by miguelTowsonCompSci** [JalIrani/tomsoft#12]
We think it would be best if we rename files when we move them because we can pull the user data and class from the DB plus we can add a datetime to the end. 

When files are being copied/moved we should be able to get the copy percentages get the data don’t worry about displaying it.

**Comment by nlicci1** [comment]
I do not think this is not feasible because there is no way to ping Google drive for status information when a new file has been added/moved.
